<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeDlg</class>
 <widget class="QDialog" name="ChangeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change</string>
  </property>
  <widget class="QLabel" name="ChangeDlg_label_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>수정 ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ChangeDlg_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>변경 할 내용 작성</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ChangeDlg_label_name">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChangeDlg_lineEdit_id">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>28</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ChangeDlg_lineEdit_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>106</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeDlg_btn_Fine">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>27</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeDlg_btn_Change">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>105</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangeDlg_btn_Close">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
